routes/wow.py

```
@@ -32,3 +33,61 @@
+
+
+@wow_bp.route("/wowoutofstock", methods=["GET", "POST"])
+def wow_outofstock_api():
+    if request.method == "GET":
+        return return_safe_html(render_template("wow_outofstock.html"))
+    elif request.method == "POST":
+        category = int(request.form.get("item_class"))
+        if category == -1:
+            include_cat = []
+        else:
+            include_cat = [category]
```

Ruff

Use ternary operator `include_cat = [] if category == -1 else [category]` instead of `if`-`else`-block

Replace `if`-`else`-block with `include_cat = [] if category == -1 else [category]`

(SIM108)